source/server-selection/server-selection.rst

```diff
@@ -885,6 +885,8 @@ to the server differently:
 - Type Mongos: the read preference is sent to the server using the rules
   for `Passing read preference to mongos`_.
 
+- Type Standalone: the read preference is ignored
```

Comment:
Each time I've considered this spec change, I get worried about SlaveOK, and then remember that we're talking about Standalone and not Single.  But it makes me wonder if we should add something for clarity like: `(note: this does not apply to Topology Type Single for other server types than Standalone)`

Reply:
That note seems like a reasonable way to preemptively avoid a misunderstanding.

I would also suggest rewording "the read preference is ignored" as "the read preference is never sent to the server" or "drivers MUST NOT send the read preference to the server" if we want to be prescriptive.

In the context of this section ("the read preference is communicated to the server"), "is ignored" makes sense but a rewording would make this clearer both here and in the changelog.

Reply:
@xdg, if you look at this change in context I think the meaning is clear, as it appears in a section titled `Topology type: Single", and there are bullets that cover all other server types.  In that context, the suggested note sounds a bit weird.  

@jmikola, I agree with your suggestion and will make it in the next patch, but I'll wait to hear back from @xdg first.

Reply:
Looking with more diff context, I agree that my additional language isn't needed, but I also agree with Jeremy about avoiding the term "ignored".

Reply:
Done